### DISCUSSION TITLE
MOBILE VIEW make the page scroll up automatically after adding the product to the cert

### DISCUSSION THREAD

**Discussion by sabadat** [breezefront/community#5] [2. Ideas]
hi i love your theme super fast no junk code, I have one issue with the breeze theme, once you added a product to the cert it displays a message but the customer needs to scroll up in order to see the message, especially on the mobile view, can u help to make the page scroll up automatically after adding the product to the cert, awaiting your support and help, 

Please to see the issue please visit this URL https://breeze.swissupdemo.com/breeze_evolution/ina-compression-short.html and make the  browser mobile view size and add the cert, you need to scroll up to see "You added Ina Compression Short to your [shopping cart]"
Regards SAB

**Comment by vovayatsyuk** [comment]
Thank you, we will think about how to improve messages visibility after ajax actions. 

As a first idea, we can make add a fixed position to all messages (make them "toast" messages). Then, add a close button so you can close it wherever you need.

Something like https://tailwindui.com/components/application-ui/overlays/notifications

**Comment by vovayatsyuk** [comment]
In the upcoming breeze release, you will have an ability to set less variable as follows:

```
@message__position: fixed;
```

To get the result shown below:

[Fixed message position](https://user-images.githubusercontent.com/306080/197731288-a232cd2e-2208-48df-a986-bba2a20451ee.webm)

**Reply by gfomengia** [reply]
> In the upcoming breeze release, you will have an ability to set less variable as follows:
> 
> ```
> @message__position: fixed;
> ```
> 
> To get the result shown below:
> 
>  ezgif.com-gif-maker.webm

Is possible to also make the message auto-disappear after a set number of seconds without clicking on anything else?

**Reply by vovayatsyuk** [reply]
Yes! Added `@message-hide__delay: 5s;` variable:

https://user-images.githubusercontent.com/306080/198326927-3c43134f-11d0-42fc-a8b1-8a3c24aef1f1.mp4
